projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -110,6 +110,26 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     if (this.resetMenuOnClose) {
       this.resetOnMenuCollapse();
     }
+    if (
+      this.featureConfigService?.isEnabled('a11yNavigationUiKeyboardControls')
+    ) {
+      this.focusOnMenuExpansion();
+    }
+  }
+
+  /**
+   * Focus on the first focusable element in the hamburger menu when the menu is opened.
+   */
+  focusOnMenuExpansion(): void {
+    this.subscriptions.add(
+      this.hamburgerMenuService?.isExpanded.pipe().subscribe((isExpanded) => {
```

Comment:
Do we need to add the empty `pipe()` here?

Reply:
nope, removed